docs/pages/smart-contracts/modular-account-v2/using-7702.mdx

```diff
@@ -68,7 +68,47 @@ const signer = LocalAccountSigner.privateKeyToAccountSigner(privateKey);
 
 Account Kit provides support for you to bring your existing custom signer or another third-party embedded wallet provider! As long as your signer supports `signAuthorization` you can connect it to Account Kit and upgrade existing EOAs to smart accounts.
 
-Follow the guide here to understand how to implement the interface: [Third Party Signers](/wallets/third-party/signers#implementing-smartaccountauthenticator-or-smartaccountsigner).
+To bring your own signer, you'll need to define a `SmartAccountSigner` interface that implements `signAuthorization`. See the details for the interface requirements [here](/wallets/third-party/signers#implementing-smartaccountauthenticator-or-smartaccountsigner). For example, you can extend a Wallet Client to include signAuthorization, if supported by the signer, like so:
+
+<Accordion title="Example">
+```tsx
+import { WalletClientSigner, type SmartAccountSigner } from "@aa-sdk/core";
+import { createWalletClient, custom } from "viem";
+import { sepolia } from "viem/chains";
+
+const embeddedWallet = window.ethereum; // or any other EIP-1193 provider
+const baseSigner = new WalletClientSigner(
+createWalletClient({
+account: embeddedWallet!.address as Hex,
+chain: sepolia
+transport: custom(embeddedWallet),
```

Comment:
Is this code missing all the indentation, or does that get added when in renders?